jujugui/static/gui/src/app/state/test-state.js

```diff
@@ -29,6 +29,10 @@ describe('State', () => {
     state: {special: {deployTarget: 'cs:trusty/kibana-15'}},
     error: null
   }, {
+    path: 'http://abc.com:123/new/?dd=trusty/kibana-15',
```

Comment:
You'll end up in a state where you will get both and the app will choose whichever one it wants to render first. That is an invalid path so I didn't feel it necessary to test it.